main.py

```diff
@@ -109,15 +112,16 @@ def get_anonymous_commits_stats(repos: list, header: dict) -> dict:
                 with open(os.path.join('repo-stats', f'{repo}.anonymous.json'), 'w') as f:
                     json.dump(response.json(), f)
 
-            print(f"{i + 1}/{len(repos)} - {repo} - {'OK' if response.status_code == 200 else 'Awaiting new data...'}")
+            print("\t" f"{i + 1}/{len(repos)} - {repo} - {'OK' if response.status_code == 200 else 'Awaiting new data...'}")
 
             if response.status_code == 403:
                 raise_rate_limited_exception()
             elif response.status_code == 200:
                 stats[repo] = sum([weekly['total'] for weekly in response.json()])
                 stats['total'] += stats[repo]
+
         else:
-            print(f'Using cache for repo {repo}')
+            print("\t" f"{i + 1}/{len(repos)} - {repo} - Using cached result...")
```

Comment:
Use only one string when two are not necessary. See also lines 115, 154, 167, 338, 483
```suggestion
            print(f"\t{i + 1}/{len(repos)} - {repo} - Using cached result...")
```

Reply:
Should be resolved in 0ee80ba